classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -444,10 +444,17 @@ function reviewRead($args, $request) {
 			$reviewAssignmentDao->updateObject($reviewAssignment);
 		}
 
+		$updateReviewNotifications = false;
 		if (!$reviewAssignment->getDateCompleted()) {
 			// Editor completes the review.
 			$reviewAssignment->setDateConfirmed(Core::getCurrentDate());
 			$reviewAssignment->setDateCompleted(Core::getCurrentDate());
+			$updateReviewNotifications = true;
+		}
+
+		$this->_updateReviewRoundStatus($reviewAssignment);
+
+		if ($updateReviewNotifications) {
```

Comment:
Is this just to avoid unnecessarily changing the notifications? I would assume that `updateNotifications` won't cause problems if called unnecessarily. (If that's right, no need to change it -- just checking.)